### PR TITLE
DebugInfo on SandboxView

### PR DESCRIPTION
**Pull request by sanyavertolet**
### What's done:
- [x] Updated CONTRIBUTING.md
- [x] Added AceModes.Python
- [x] Added short debug info alert on SandboxView
- [x] Added debugInfo modal on SandboxView
- [x] Added fetch-debug-info button in codeEditorComponent

### How it looks:
https://user-images.githubusercontent.com/35039155/194892646-7e415093-4e95-4f7b-8fa7-983161081bda.mov

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt [hunk 1]
```diff
@@ -280,7 +361,8 @@ class SandboxView : AbstractView<SandboxViewProps, SandboxViewState>(true) {
             |}
         """.trimMargin()
         private val setupShExample = """
-            |python3.10 -m pip install pylint
+            |# Here you can add some additional commands required to run your tool e.g.
+            |# python3.10 -m pip install pylint
```
Comment (petertrr) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt:365]: Why do we have a fixed version of python here? Maybe simply `python` would be more generic?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt [hunk 2]
```diff
@@ -146,27 +186,52 @@ class SandboxView : AbstractView<SandboxViewProps, SandboxViewState>(true) {
...
+                    is Crash -> "danger"
+                    else -> "info"
+                }
+                div {
+                    className = ClassName("alert alert-$alertStyle alert-dismissible fade show")
+                    role = AriaRole.alert
+                    div {
+                        displayTestResultDebugInfoStatus(debugInfo)
+                        a {
+                            href = "/#/sandbox"
```
Comment (petertrr) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt:213]: Could you try to use `Link { to }` here? We discussed that this is the preferred approach for react-router

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt [hunk 3]
```diff
@@ -146,27 +186,52 @@ class SandboxView : AbstractView<SandboxViewProps, SandboxViewState>(true) {
...
+                            +"See more details..."
+                            onClick = {
```
Comment (petertrr) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt:215]: Oh, wait, is this an `<a>` with overridden `onClick`? Then why do we need `href` attribute?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt (hunk 1 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt:365]: Sure

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt (hunk 2 above)
Comment (Cheshiriks) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt:213]: Can use here Navigate?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt (hunk 3 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt:215]: Thought it is required to keep `a`'s styles. Will remove it.

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt (hunk 2 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt:213]: Seems like I do not need this `href` so I will remove it and that's all

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt (hunk 1 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt]: Done

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt (hunk 3 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt:214]: Oh we still need it. Take a look above

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt (hunk 2 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt]: Removed `href` and marked the text In a different way - `strong` and `italic`

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt (hunk 3 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt:223]: Removed `href` and marked the text In a different way - `strong` and `italic`

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt (hunk 2 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt]: Removed `href` and marked the text In a different way - `strong` and `italic`

**Review by Cheshiriks — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt (hunk 3 above)
Comment (petertrr) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt:222]: Is `button` with class `btn-link` the same as you want?

Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt:215]: Done
Thanks for your brilliant idea!

**Pull request merged**